src/wazuh_db/wdb_parser.c

```diff
@@ -5000,17 +4999,27 @@ int wdb_parse_global_get_agents_by_connection_status(wdb_t* wdb, char* input, ch
     }
     connection_status = next;
 
-    wdbc_result status = wdb_global_get_agents_by_connection_status(wdb, last_id, connection_status, &out);
+    // Execute command
+    wdbc_result status = WDBC_UNKNOWN;
+    cJSON* result = wdb_global_get_agents_by_connection_status(wdb, last_id, connection_status, &status);
+    if (!result) {
+        mdebug1("Error getting agents to be disconnected from global.db.");
```

Comment:
Typo: debug message "_Error getting agents to be disconnected ..._" inside **wdb_parse_global_get_agents_by_connection_status()**